src/commands/api/update.js

```diff
@@ -9,11 +9,18 @@ const { getResponseContent } = require('../../support/command/handle-response')
 
 class UpdateAPICommand extends UpdateCommand {
 
-  async updateApi({ owner, name, version, flags, isPrivate, visibility }) {
+  async updateApi(owner, name, version, flags) {
+    const queryParams = { version }
 
+    if (flags.visibility) {
+      queryParams['isPrivate'] = flags.visibility !== 'public'
+      this.logCommandSuccess = this.setSuccessMessage('ApiUpdateVisibility')
+    }
+
+    const visibility = queryParams['isPrivate'] ? 'private' : 'public'  
```

Comment:
Can we just pass `visibility: flags.visibility` to the `logCommandSuccess` instead of doing this? 
Visibility only used if it isn't undefined anyway.